widgets/GoogleSheets-batchsink.json

```diff
@@ -234,7 +261,33 @@
     {
       "name": "Authenticate with OAuth2",
       "condition": {
-        "expression": "authType == 'oAuth2'"
+        "property": "authType",
+        "operator": "equal to",
+        "value": "oAuth2"
```

Comment:
why this change?

Reply:
this filter is there in all 4 plugins, made them similar